app/Models/Extensions/SizeVariants.php

```diff
@@ -178,10 +178,11 @@ public function getThumb(): ?SizeVariant
 	 *                                size variant shall point to
 	 * @param int    $width           the width of the size variant
 	 * @param int    $height          the height of the size variant
+	 * @param int    $filesize        the filesize of the size variant
 	 *
 	 * @return SizeVariant The newly created and persisted size variant
 	 */
-	public function create(int $sizeVariantType, string $shortPath, int $width, int $height): SizeVariant
+	public function create(int $sizeVariantType, string $shortPath, int $width, int $height, int $filesize): SizeVariant
```

Comment:
```suggestion
	public function create(int $sizeVariantType, string $shortPath, int $width, int $height, int $filesize = 0): SizeVariant
```

Reply:
Done with -1 default value (see migration and command).